test/unit/services/platform-test.js

```diff
@@ -358,10 +357,16 @@ describe('Platform', function() {
           expect(result).to.be.equal(248005160960);
         });
       });
+
+      it('should return NaN if drive not found', function() {
+        sandbox.stub(child_process, 'exec').yields(undefined, "");
+        let location = 'd:\\DevelopmentSuite';
+        return Platform.getFreeDiskSpace(location).then((result) => {
+          expect(result).to.be.NaN;
+      });
```

Comment:
I think you forgot to close one parentheses combo after the then callback. There's a syntax error now.